kubernetes/resource_kubernetes_endpoints_test.go

```diff
@@ -128,6 +131,11 @@ func TestAccKubernetesEndpoints_basic(t *testing.T) {
 									Port:     443,
 									Protocol: api.ProtocolTCP,
 								},
+								{
```

Comment:
I'm not really satisfied with this.
In the corresponding test configuration, that port is defined last.
But putting the port last in this list, it triggers the following error:

```
--- FAIL: TestAccKubernetesEndpoints_basic (0.60s)                                                                                                                                      
    testing.go:568: Step 1 error: Check failed: 1 error occurred:           
                * Check 31/31 error: Unexpected output from expander: mismatch (-want +got):                                                                                            
        {[]v1.EndpointSubset}[1].Ports[1].Name:                        
                -: "httpstransport2"                                            
                +: "aaaa"                                                
        {[]v1.EndpointSubset}[1].Ports[1].Port:                                   
                -: 444                                                         
                +: 442                            
        {[]v1.EndpointSubset}[1].Ports[2].Name:                                                                                                                                         
                -: "aaaa"                                               
                +: "httpstransport2"                                   
        {[]v1.EndpointSubset}[1].Ports[2].Port:                                 
                -: 442                                                                                                                                                                  
                +: 444                                                    
```

IMO, the test case should not make assumptions about that list order now that it is actually a set on terraform's end.